docs/mine/mitigating-dos.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Mitigation of DoS attacks
+---
+
+# Mitigation of DoS attacks
+
+Any service accessible to the Internet is exposed to a multitude of potential attacks that can be done intentionally (malicous actor), or indirectly (network scans, enumeration, etc). This a challenge not exclusive to Filecoin and it is a very well developed area with many organizations offering all kinds of services to protect from it. In this page, you will find some recommendations, but ultimately you should explore other solutions that are available to you.
+
+## Port based protection
+
+When running your Miner from a Public IP address, every single port on your machine will be accessible from the Internet by default. We recommend that you block all incoming traffic except for the ports that are essential, namely SSH and the Lotus listen addresses.
+
+First, [set a static port for your Miner using the guide](/mine/setting-a-static-port)
```

Comment:
```suggestion
First, [set a static port for your Miner using the guide]./setting-a-static-port.md)
```

This is what @cwaring is looking for. Again, suggesting here so folks can see what relative links should look like, with the `.md` extension included :)